bin/launch_utils.py

```diff
@@ -26,4 +29,45 @@ def import_class_from_file(file_path: str) -> type:
 
 def get_experiment(experiment_name: str) -> object:
     experiment_object = getattr(exp, experiment_name)()
-    return experiment_object
\ No newline at end of file
+    return experiment_object
+
+
+def memory_split_for_ray_init(memory_str:  Union[str, None]) -> Tuple[float, float]:
+    """
+    compute the memory requirements for ray init. 
```

Comment:
Process the input memory value into the right unit and allocates 30% for overhead and 70% for tuning.